src/test/java/org/jboss/threads/QueuelessViewExecutorTest.java

```diff
@@ -80,7 +123,7 @@ public void testShutdownNow() throws InterruptedException {
     public void testShutdownNow_immediatelyAfterTaskIsSubmitted() throws InterruptedException {
```

Comment:
This test would fail on the queued implementation without the fix provided by this PR. The queueless implementation already does what I've proposed for the queued implementation.